validator/tests/test_checks.py

```diff
@@ -146,21 +146,21 @@ def test_expected_groups(self):
         assert pattern_found["signal"] == "signal_signal"
 
 
-class TestCheckBadGeoId:
+class TestCheckBadGeoIdFormat:
     params = {"data_source": "", "span_length": 0,
               "end_date": "2020-09-02", "expected_lag": {}}
```

Comment:
```suggestion
              "end_date": "2020-09-02", "expected_lag": {},
              "validator_static_file_dir": "../static"}
```

Default path to `static` dir doesn't work for tests. Specify explicitly.